.gitignore

```diff
@@ -6,5 +6,6 @@ __pycache__/
 .DS_Store
 .tm_properties
 
-# Ignore optional build directory
+# Ignore optional build / cache directory
 /build
+/downloads_cache
```

Comment:
The standard structure is to have the downloads cache inside the build directory. I don't want to support multiple different structures since it can get confusing. I also like to keep .gitignore files as short and simple as possible to encourage cleaner file structures. If you want to propose a new standard structure, please let me know.

Reply:
The build script is not working as expected doh
I will check this out

Reply:
Oh, I see. I never tested that script, so feel free to fix it. It's a recent addition to deprecate batch scripts.